haggis.js

```diff
@@ -369,6 +372,12 @@ function renderStatsAsHtmlString(tableId, stats) {
     output += `    <th>${player1}</th>\n`;
     output += `    <th>${player2}</th>\n`;
     output += "  </thead>\n";
+    output += "  <tr>\n";
+    output += "    <td>Starting Score </td>\n";
+    output += `    <td>${round.startingScore[player2] ? round.startingScore[player1] : 0}</td>\n`;
```

Comment:
I think you can use the `??` operator here like: `round.startingScore[player1] ?? 0`